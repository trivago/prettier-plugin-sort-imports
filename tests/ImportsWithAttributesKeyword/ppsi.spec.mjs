run_spec(__dirname, ["typescript"], {
    importOrder: ['^@core/(.*)$', '^@server/(.*)', '^@ui/(.*)$', '^[./]'],
    importOrderSeparation: false,
    importOrderSideEffects: false,
    importOrderParserPlugins: ['typescript'],
    importOrderImportAttributesKeyword: 'with',
});
