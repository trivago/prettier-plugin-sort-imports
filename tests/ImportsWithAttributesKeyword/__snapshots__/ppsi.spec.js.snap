// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`imports-with-attributes-keyword.ts - typescript-verify: imports-with-attributes-keyword.ts 1`] = `
// I am top level comment in this file.
import thirdParty0 from "third-party0";
import something3 from "@core/something3";
import thirdDisco0 from "third-disco0";
import otherthing3 from "@core/otherthing3";
import { a } from "b" with { type: "json" };

import anotherSameLevelRelativePath3 from "./anotherSameLevelRelativePath3";
import something0 from "@core/something0";

import { b } from "r" with { type: "json" };


function add(a,b) {
  return a + b;
}~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
// I am top level comment in this file.
import { a } from "b" with { type: "json" };
import { b } from "r" with { type: "json" };
import thirdDisco0 from "third-disco0";
import thirdParty0 from "third-party0";
import otherthing3 from "@core/otherthing3";
import something0 from "@core/something0";
import something3 from "@core/something3";
import anotherSameLevelRelativePath3 from "./anotherSameLevelRelativePath3";

function add(a, b) {
    return a + b;
}

`;
